Bottom: e001bbbac130db134ee0affd89d76ec75afb1c29
Top:    7c31b4dd2ce03e6852cf6c2d1f10dccdcc354c17
Author: Robert Berger <robert.berger@ReliableEmbeddedSystems.com>
Date:   2021-01-18 16:46:30 +0200

resolved collision between perf and trace-cmd

Signed-off-by: Robert Berger <robert.berger@ReliableEmbeddedSystems.com>


---

diff --git a/meta-oe/recipes-kernel/trace-cmd/trace-cmd_2.9.1.bb b/meta-oe/recipes-kernel/trace-cmd/trace-cmd_2.9.1.bb
index d39afff8e..717bb6784 100644
--- a/meta-oe/recipes-kernel/trace-cmd/trace-cmd_2.9.1.bb
+++ b/meta-oe/recipes-kernel/trace-cmd/trace-cmd_2.9.1.bb
@@ -14,4 +14,15 @@ do_install() {
        oe_runmake etcdir=${sysconfdir} DESTDIR=${D} install
 }
 
-FILES_${PN} += "${libdir}/traceevent/plugins"
+# ${libdir}/traceevent/plugins clashes with package perf
+# except for plugin_blk.so it seems
+# so we put it in it's own package
+
+# FILES_${PN}-plugins = "${libdir}/traceevent/plugins"
+
+FILES_${PN} += "${libdir}/traceevent/plugins/plugin_blk.so"
+
+PACKAGES =+ "${PN}-trcevplug"
+FILES_${PN}-trcevplug += " \
+     ${libdir}/traceevent/plugins \
+"
